app/components/finance/npq/total_payment_row.html.erb

```diff
@@ -2,5 +2,5 @@
   <td scope="row" class="govuk-table__cell govuk-!-font-weight-bold govuk-!-width-one-quarter"><%= t("finance.total_payment") %></td>
   <td scope="row" class="govuk-table__cell govuk-table__cell--numeric govuk-!-width-one-quarter"/>
   <td scope="row" class="govuk-table__cell govuk-table__cell--numeric govuk-!-width-one-quarter"/>
-  <td scope="row" class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold govuk-!-width-one-quarter"><%= number_to_pounds(total) %></td>
+  <td scope="row" class="govuk-table__cell govuk-!-font-weight-bold govuk-table__cell--numeric govuk-!-width-one-quarter"><%= number_to_pounds(npq_total_payment(breakdown) + npq_total_vat(breakdown, lead_provider)) %></td>
```

Comment:
Could we calculate this in the component class? Helps keeping the views with as little logic as possible.

Reply:
Now updated 👍